.circleci/config.yml

```
@@ -117,6 +136,14 @@ workflows:
             branches:
               only: master
       - test
+      - publish:
+          requires:
+            - test
+          filters:
+            tags:
+              only: /^v.*/
+            branches:
+              ignore: /.*/
```

should this not be only master?
```suggestion
              only: master
```

---

I thought that, as well. Will change